department-of-veterans-affairs/va.gov-cms
# VACMS-5785: Add confirm step to vbo.

## Description
See #5785

## Testing done
Visual / behat

## QA steps
- As an administrator, go to `admin/content/bulk` , select a few items, and perform various vbo ops.
- Visually verify a confirmation step.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
